src/dxtbx/model/goniometer.h

```diff
@@ -36,31 +36,45 @@ namespace dxtbx { namespace model {
    * A class to represent the rotation axis for a standard rotation
    * geometry diffraction data set.
    *
-   * The rotation axis assumed to have its origin at the origin of the
+   * The rotation axis is assumed to have its origin at the origin of the
    * laboratory coordinate system. The rotation axis vector is normalized
-   * when set using either the constructor or the rotation axis 'setter'
+   * when set using either the constructor or the rotation axis 'setter'.
    *
-   * The fixed rotation matrix, F, represents the additional fixed rotation of
-   * the sample attached to the rotating axis - for example the effects of
-   * kappa and phi for an omega scan on a kappa goniometer. A second rotation
-   * matrix, S, may be applied which is used to align the rotation axis,
-   * for example the omega & kappa axes for a phi scan. These rotation matrices
-   * should be applied as:
+   * In the conventional crystallographic formalism, the goniostat rotation
+   * operator R represents the rotation of the sample mount and relates the
+   * sample reference frame to the lab frame.  R is calculated from the
+   * recorded angular positions of each axis, combined with prior knowledge
+   * of the orientation in the lab frame of each axis when the goniometer is
+   * at its zero datum.
    *
-   *  A = [S][R][F][U][B]
+   * In the DXTBX goniometer model, we define a finer-grained decomposition
+   * of R to represent the different axes of a general goniometer, which may
+   * have one single axis or multiple axes.  This model is intended to
```

Comment:
To answer these in turn:
- I think you're right, I could probably be clearer in distinguishing between motors of a goniometer (which is common parlance are referred to as axes, as in '_φ_ axis') and the axis of rotation of those motors.  Perhaps if I make that distinction more clearly, it might be a little clearer.  Words like 'setting' and 'fixed' were chosen by someone else before my time though, and I've tried not to change that for fear of unintended mix-ups elsewhere.
- Noted, I'll have a go.
- I've used 'static' here to refer to those motors of a goniometer (and their associated rotation operators) that are not actuated during a scan, though the motor may move in the lab frame by virtue of the motion of a parent motor.  So by 'static', I mean that the operator itself is not changing, dS/dt = dF/dt = 0.  The DXTBX goniometer model is predicated on only a single motor moving during a scan and we represent that motor's rotation operation as _R'_, so by contrast dR'/dt ≠ 0 and I refer to _R'_ as 'scanning'.  So, for example, for an _ω_ scan on a three-axis κ-geometry goniometer, _ω_ is 'scanning' and I refer to _κ_ & _φ_ as 'static' because their motors are fixed (and hence F does not change), even though the movement of _ω_ causes the entire _κ_ and _φ_ apparatus to move in the lab frame, because they are mounted on _ω_.  This movement is encapsulated in the changing _R'_.
- I'm afraid I didn't choose the terminology 'setting' and 'fixed', and I'm reluctant to go through doing a root-and-branch rewrite of the method and attribute names in the code, as well as the docs.  I only started this exercise in an effort to make sense of the code in its current form.